‘class tinyspline::BSpline’ has no member named ‘data’

I get this error while trying to build LibreCad.
```
/pathtolibrecad/LibreCAD_3/lckernel/cad/geometry/geospline.cpp: In member function ‘void lc::geo::Spline::populateCurve()’:
/pathtolibrecad//LibreCAD_3/lckernel/cad/geometry/geospline.cpp:103:59: error: ‘class tinyspline::BSpline’ has no member named ‘data’
  103 |     int nbBeziers = ts_bspline_num_control_points(beziers.data()) / splineCurve.order();
      |                                 
```
Also, I had an error prior to that with the command git submodule update --recursive --remote

```
Submodule path 'lckernel/tinyspline': checked out '2cd630a171e9ab2bd649994954b46eb47e2248a6'
fatal: Needed a single revision
Unable to find current origin/master revision in submodule path 'third_party/kaguya'

```
I'm on Xubuntu 20.04.

--- a/lckernel/cad/geometry/geospline.cpp
+++ b/lckernel/cad/geometry/geospline.cpp
@@ -100,7 +100,7 @@ void Spline::populateCurve() {
 
     auto beziers = splineCurve.toBeziers();
 
-    int nbBeziers = ts_bspline_num_control_points(beziers.data()) / splineCurve.order();
+    int nbBeziers = beziers.numControlPoints() / splineCurve.order();
     int nbCoordinate = splineCurve.order() * splineCurve.dimension();
 
     auto controlPoints = beziers.controlPoints();

